agent/util-scripts/pbench-collect-sysinfo

```diff
@@ -17,7 +17,7 @@ pbench_bin="`cd ${script_path}/..; /bin/pwd`"
 group=default
 dir="/tmp"
 sysinfo_opts_default=( block libvirt kernel_config sos topology )
-sysinfo_opts=${sysinfo_opts_default[*]}
+sysinfo_opts="${sysinfo_opts_default[*]} ara"
```

Comment:
@chaitanyaenr, perhaps @atheurer remembers differently, but we discussed the idea that all collect-sysinfo options should be on by default, and just check whether or not they can be collected at the time of collection, not giving an error if the data is not available.

So it would seem that with the check below, having this be collected by default is safe.